components/inputs/sass/text.scss

```diff
@@ -27,7 +27,7 @@
 	width: 100%;
 	&::placeholder,
 	&::-ms-input-placeholder {
-		color: $d2l-color-mica;
+		color: $d2l-color-galena;
```

Comment:
This will likely impact stuff in MVC/legacy.

Reply:
Confirmed with Jeff that this is fine and we will also be trying to clean up those usages.